.github/workflows/deploy.yml

```diff
@@ -1,360 +1,93 @@
-  #         aws-region: ${{ secrets.AWS_DEFAULT_REGION }}
-
-  #     - name: output account id
-  #       id: output_account_id
-  #       run: |
-  #         #!/bin/bash
-  #         AWS_ACCOUNT_ID=$(aws sts get-caller-identity --query Account --output text)
-  #         echo "Current Account ID: ${AWS_ACCOUNT_ID}"
-
-  #     - name: Get Github Actions CIDR Blocks
-  #       id: get-gha-cidrs
-  #       shell: bash
-  #       run: |
-  #         #!/bin/bash
-  #         GHA_RESP=$(curl --header 'authorization: Bearer ${{ secrets.GITHUB_TOKEN }}' https://api.github.com/meta)
-  #         echo "Response for GHA runner CIDR blocks:  $GHA_RESP"
-  #         IPV4_CIDR_ARR=($(echo $GHA_RESP | jq -r '.actions | .[]' | grep -v ':'))
-  #         GHA_CIDRS_IPV4=$(echo $(IFS=" "; echo ${IPV4_CIDR_ARR[*]}))
-  #         echo "GHA_CIDRS_IPV4=$GHA_CIDRS_IPV4" >> $GITHUB_OUTPUT
-
-  #     - name: Generate IP Set Name
-  #       id: gen-ip-set-name
-  #       run: |
-  #         #!/bin/bash
-  #         STAGE_GH_IPSET_NAME=$branch_name-gh-ipset
-  #         echo "Github IP Set name:  $STAGE_GH_IPSET_NAME"
-  #         echo "STAGE_GH_IPSET_NAME=$STAGE_GH_IPSET_NAME" >> $GITHUB_OUTPUT
-
-  #     - name: Fetch AWS IP Set Metadata
-  #       id: fetch-ip-set-info
-  #       run: |
-  #         #!/bin/bash
-  #         # Fetch AWS IP set ARNs using AWS CLI and store them in a variable
-  #         AWS_IP_SET_INFO=$(aws wafv2 list-ip-sets --scope=CLOUDFRONT)
-  #         echo "Outputting AWS IP Set Info:  ${AWS_IP_SET_INFO}"
-  #         # Store the IP set ARNs in an output variable using GITHUB_OUTPUT
-  #         IPSET_NAME=${{ steps.gen-ip-set-name.outputs.STAGE_GH_IPSET_NAME }}
-  #         IPSET=$(jq '.IPSets | map(select(.Name == "'${IPSET_NAME}'")) | .[]' <<< ${AWS_IP_SET_INFO})
-  #         [ -z "$IPSET" ] && echo "IP Set with name ${IPSET_NAME} was not located.  Exiting..." && exit 1
-  #         echo "IP Set metadata:  ${IPSET}"
-  #         #Get Values from the IP SET
-  #         IPSET_ID=$(jq -r '.Id' <<< ${IPSET})
-  #         echo "IPSET_ARN=$IPSET_ARN" >> $GITHUB_OUTPUT
-  #         echo "IPSET_NAME=$IPSET_NAME" >> $GITHUB_OUTPUT
-  #         echo "IPSET_ID=$IPSET_ID" >> $GITHUB_OUTPUT
-
-  #     - name: Update IP Set
-  #       id: update-ip-set
-  #       run: ./.github/wafController.sh set ${{ steps.fetch-ip-set-info.outputs.IPSET_NAME }} ${{ steps.fetch-ip-set-info.outputs.IPSET_ID }} ${{ steps.get-gha-cidrs.outputs.GHA_CIDRS_IPV4 }}
-  #       env:
-  #         AWS_RETRY_MODE: adaptive
-  #         AWS_MAX_ATTEMPTS: 10
-
-  #   outputs:
-  #     ipset_name: ${{ steps.fetch-ip-set-info.outputs.IPSET_NAME }}
-  #     ipset_id: ${{ steps.fetch-ip-set-info.outputs.IPSET_ID }}
-
-  e2e-test:
-    name: E2E Integration Tests
-    if: ${{ always() && !cancelled() && needs.deploy.result == 'success' && github.ref_name != 'production' && !startsWith(github.ref_name, 'skipci') && !startsWith(github.ref_name, 'code-json-') }}
```

Comment:
we should keep this

Reply:
Do you mean the if statement or the integration tests? The integration tests are still around and being run. The if statement is also still there but simplified by the branch patterns making parts of it no longer needed.